com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,95 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+    private const val EMPTY_STRING = ""
```

Comment:
I'd say an empty string is not really a constant so we don't need to declare that here. Inline we can just do `?: ""` wherever needed.

Reply:
Yep